NOTICE.txt

```diff
@@ -0,0 +1,442 @@
+
+synthetics-recorder
+Copyright (c) 2017-present, Elasticsearch BV
```

Comment:
Re-running the script updates this to `2021-present`, so we might want to overwrite this with the latest output of the script.

Reply:
Thanks, updated!